source/documentation/managing_apps/ssh.erb

```diff
@@ -2,104 +2,98 @@
 
 ## Connecting with SSH
 
-When you deploy an app to GOV.UK PaaS, it runs in a container, which is like a lightweight Linux virtual machine. Each app runs in its own isolated container.
+When you deploy an app to GOV.UK PaaS, that app runs in a [container](https://www.cloudfoundry.org/containers/).
 
-Sometimes, it can be useful to connect directly to the container with SSH. You would usually only do this to get information for troubleshooting purposes, for example, if you can't work out what is happening with your app using the `cf logs` and `cf events` commands described in the [Logs](/monitoring_apps.html#logs) section.
+If you use a [standard buildpack](responsibility_model.html#standard-buildpack-responsibilities), you can connect directly to an app’s container using SSH.
 
-If you do run commands which will change the container temporarily, it's a good idea to restart the app afterwards.
+Connecting using SSH gives you higher-level permissions to change an app's container than running [`cf logs` and `cf events`](/monitoring_apps.html#logs) on that container.
```

Comment:
> SSH gives you higher-level permissions

I'd say higher-level access here, since it's not a question of whether they have permission to change things from the outside. They simply don't have access to do it at all.